index.css

```diff
@@ -222,10 +222,20 @@ h2{
 	text-align: center;
 }
 
+@media only screen and (max-width:1140px){
+	main{
+		max-width: calc(var(--max-content-width)*(2/3));	
+		align-items: center;
+		margin:0 auto;
+	}
+
+}
+
 @media only screen and (max-width: 746px) {
 	section{
-		padding-left: 
-		justify-content: center;
+		padding-left: 20%;
+		padding-right: 20%;
```

Comment:
Why?

Reply:
> it doesn't look good when 4 are on the first line and only one on the second

This ensures the issue you raised her.

Reply:
It maybe does on your device but doesn't on mine:
![Screenshot from 2019-10-06 13-10-46](https://user-images.githubusercontent.com/12966757/66268160-c907f580-e83a-11e9-8858-cd2c1b0906aa.png)

You can't just add some magic margin and expect it to work for anyone but you.